dcsazure_AzureSQL_to_AzureSQL_prof_pl/README.md

```diff
@@ -0,0 +1,80 @@
+# dcsazure_AzureSQL_to_AzureSQL_prof_pl
+## Delphix Compliance Services (DCS) for Azure - AzureSQL to AzureSQL Profiling Pipeline
+
+This pipeline will perform automated sensitive data discovery on your AzureSQL Instance.
+
+### Prerequisites
+1. Configure the hosted metadata database and associated Azure SQL service (version `V2024.01.01.0`+).
+2. Configure the DCS for Azure REST service.
+3. Configure the AzureSQL linked service.
+
+### Importing
+There are several linked services that will need to be selected in order to perform the profiling of your AzureSQL
+instance.
+
+These linked services types are needed for the following steps:
+
+
+`AzureSQL` (source) - Linked service associated with unmasked AzureSQL data. This will be used for the following
+steps:
+* Schema Discovery From AzureSQL (Copy data activity)
+* dcsazure_AzureSQL_to_AzureSQL_source_ds (AzureSQL dataset)
+* dcsazure_AzureSQL_to_AzureSQL_prof_df/AzureSQLSource1MillRowDataSampling (dataFlow)
+
+`Azure SQL` (metadata) - Linked service associated with your hosted metadata store. This will be used for the following
+steps:
+* dcsazure_AzureSQL_to_AzureSQL_metadata_prof_ds (Azure SQL Database dataset),
+* dcsazure_AzureSQL_to_AzureSQL_prof_df/MetadataStoreRead (dataFlow),
+* dcsazure_AzureSQL_to_AzureSQL_prof_df/WriteToMetadataStore (dataFlow)
+
+`REST` (DCS for Azure) - Linked service associated with calling DCS for Azure. This will be used for the following
+  steps:
+* dcsazure_AzureSQL_to_AzureSQL_prof_df (dataFlow)
+
+### How It Works
+
+* Schema Discovery From AzureSQL
+  * Query metadata from AzureSQL `information_schema` to identify tables and columns in the AzureSQL instance
+* Select Discovered Tables
+  * After persisting the metadata to the metadata store, collect the list of discovered tables
+* For Each Discovered Table
+  * Call the `dcsazure_AzureSQL_to_AzureSQL_prof_df` data flow
+
+```mermaid
+sequenceDiagram
+    participant AzureSQL as Azure SQL Database
+    participant SchemaDiscovery as Schema Discovery
+    participant MetadataStore as Metadata Store
+    participant DataFlow as dcsazure_AzureSQL_to_AzureSQL_prof_df
+
+
+    SchemaDiscovery->>AzureSQL: Query metadata (information_schema)
+    AzureSQL-->>SchemaDiscovery: Return metadata (tables and columns)
+
+    SchemaDiscovery->>MetadataStore: Persist metadata
+    MetadataStore-->>SchemaDiscovery: Confirmation
+
+    SchemaDiscovery->>MetadataStore: Collect discovered tables
+    MetadataStore-->>SchemaDiscovery: Return list of tables
+
+    SchemaDiscovery->>SchemaDiscovery: For Each Discovered Table
+    loop For each table
+        SchemaDiscovery->>DataFlow: Call data flow (dcsazure_AzureSQL_to_AzureSQL_prof_df)
+        DataFlow-->>MetadataStore: Store profiling results
+    end
+```
```

Comment:
[todo] Need to remove it. Created another story to add these mermaid diagrams to all README files. https://delphix.atlassian.net/browse/DAAS-9542